services/web-app/components/article-card/article-card.module.scss

```
@@ -0,0 +1,86 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/type' as type;
+@use '@carbon/react/scss/motion' as motion;
+
+.article-card {
+  background: theme.$layer-01;
+  transition: background motion.$duration-fast-01;
+
+  &:hover {
+    background: theme.$layer-hover-01;
+  }
+}
+
+//overide carbon tile, custom styles for article card
+.article-card :global(.cds--tile) {
+  position: relative;
+  height: 100%;
+  padding: 0;
+  background: transparent;
+  text-decoration: none;
+}
+
+.tile {
+  padding: spacing.$spacing-05 25% spacing.$spacing-05 spacing.$spacing-05;
+}
+
+.title {
+  @include type.type-style('body-short-02');
```

```suggestion
  @include type.type-style('body-compact-02');
```
should this be `body-compact-02` instead in v11?